docs/UserGuide.md

```diff
@@ -241,9 +242,10 @@ List all contacts and their details.
 ### Checking who is free : `whoisfree`
 View all persons that are available on the specified day and time.
 
-Format: `whoisfree DAY:TIME`
-- `DAY` is from Mon-Sun.
-- `TIME` is 24-hour time format.
+Format: `whoisfree TIMESTAMP`
```

Comment:
I feel without the example it's a little unclear.. but w the example is ok ish??